atc/steps.go

```diff
@@ -30,7 +30,10 @@ func (step *Step) UnmarshalJSON(data []byte) error {
 
 		err := step.ParseJSON(data)
 		if err != nil {
-			return err
+			return MalformedStepError{
+				StepType: "get",
```

Comment:
Should this be `s.Key`?

```suggestion
				StepType: s.Key,
```

Reply:
Hahaha. Yes. (Hey, it passed the tests!)